server/elasticsearch/elasticsearch.ts

```diff
@@ -2,3 +2,45 @@ import { Client } from 'elasticsearch';
 import config from '../config';
 
 export default new Client(config.elasticsearch);
+
+export type EsReportSource = {
```

Comment:
I don't think we can make any assumptions about the structure of the data.  We might have to settle for using `any`.  The reason is the structure will be different based on permissions, custom columns, and other things we might add in the future such as muster information.